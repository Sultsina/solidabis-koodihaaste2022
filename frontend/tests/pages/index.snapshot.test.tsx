import renderer from "react-test-renderer";
import Home from "../../pages/index";

jest.mock("@mui/material/Button", () => "button");
jest.mock("@mui/material/TextField", () => "textField");

it("renders correctly without input", () => {
  const tree = renderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
