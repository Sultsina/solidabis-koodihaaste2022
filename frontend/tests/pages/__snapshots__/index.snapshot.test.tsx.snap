// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without input 1`] = `
.c0 {
  margin-top: 3em;
  display: grid;
  justify-items: center;
}

.c1 {
  margin-bottom: 2em;
}

.c1 > div {
  place-self: center;
}

.c2 > div {
  place-self: center;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <textField
      data-testid="city"
      label="What city are you in?"
      onChange={[Function]}
      variant="standard"
    />
  </div>
  <div
    className="c2"
  >
    <button
      data-testid="select-city"
      disabled={true}
      onClick={[Function]}
      variant="contained"
    >
      Select
    </button>
  </div>
</div>
`;
